cax-ap-cam/cax_ap_cam/const.py

```diff
@@ -0,0 +1,82 @@
+from siriuspy.envars import VACA_PREFIX as _vaca_prefix
+from siriuspy import util as _util
+
+
+class Constants:
+
+    def __init__(self, devname=''):
+        self.ioc_prefix = _vaca_prefix + ('-' if _vaca_prefix else '')
+        self.ioc_prefix += devname + ':'
+
+        self.set_database()
+
+    def _get_intensity_saturated_db(self):
+        sufix = '-RB'
+        return {
+            'Intensity_Min' + sufix: {
+                'type': 'int', 'unit': 'intensity'
+            },
+            'Intensity_Max' + sufix: {
+                'type': 'int', 'unit': 'intensity'
+            },
+            'Intensity_Avg' + sufix: {
+                'type': 'float', 'unit': 'intensity'
+            },
+            'Intensity_Sum' + sufix: {
+                'type': 'int', 'unit': 'intensity'
+            },
+            'Is_Saturated' + sufix: {
+                'type': 'int'
```

Comment:
I think we can remove those "_" in PV names: `Intensity_Min` -> `IntensityMin`

Reply:
done